.changeset/slimy-islands-end.md

```diff
@@ -0,0 +1,14 @@
+---
+"@builder.io/sdk": patch
+"@builder.io/react": patch
+"@builder.io/sdk-angular": patch
+"@builder.io/sdk-react-nextjs": patch
+"@builder.io/sdk-qwik": patch
+"@builder.io/sdk-react": patch
+"@builder.io/sdk-react-native": patch
+"@builder.io/sdk-solid": patch
+"@builder.io/sdk-svelte": patch
+"@builder.io/sdk-vue": patch
+---
+
+Fix: meta.componentsUsed was incorrectly omitted by default when using fetchOneEntry.
```

Comment:
- when writing code in changesets its best to use backquotes \`\`, e.g. `meta.componentsUsed`
- `fetchOneEntry` is the name for Gen2 specifically, but this changeset is for gen1 as well

```suggestion
Fix: correctly set default value for `meta.componentsUsed` in Content API calls.
```